en/config_mc/position_mode_smooth_vel.md

```diff
@@ -0,0 +1,65 @@
+# Jerk-limited Modes
+
+This type of setpoints generator is active in:
```

Comment:
As above, would be good to have a paragraph explaining the broad aims of the config, contrasted against sports/smooth (I am assuming these are mutually exclusive).